airflow/utils/cli.py

```diff
@@ -107,6 +113,10 @@ def wrapper(*args, **kwargs):
             finally:
                 metrics['end_datetime'] = datetime.utcnow()
                 cli_action_loggers.on_post_execution(**metrics)
+                if verbose:
+                    root_logger.setLevel(logging.NOTSET)
+                    for handler in root_logger.handlers:
+                        handler.setLevel(logging.NOTSET)
```

Comment:
You are right; we don't need to unset, and users are not expected to call this from python. 
Also, since it's exiting, we don't need to track the config. I was looking at a similar unsetting code down the module when I implemented that but it's actually not necessary